perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -376,9 +376,16 @@ sub inspect : Private {
     $c->stash->{report_meta} = { map { 'x' . $_->{name} => $_ } @{ $c->stash->{problem}->get_extra_fields() } };
 
     if ($c->cobrand->can('council_area_id')) {
-        my $priorities_by_category = FixMyStreet::App->model('DB::ResponsePriority')->by_categories($c->cobrand->council_area_id, @{$c->stash->{contacts}});
+        my $priorities_by_category = FixMyStreet::App->model('DB::ResponsePriority')->by_categories($c->cobrand->council_area_id, $c->stash->{contacts});
         $c->stash->{priorities_by_category} = $priorities_by_category;
-        my $templates_by_category = FixMyStreet::App->model('DB::ResponseTemplate')->by_categories($c->cobrand->council_area_id, @{$c->stash->{contacts}});
+        my $templates_by_category = FixMyStreet::App->model('DB::ResponseTemplate')->by_categories(
+            $c->cobrand->council_area_id,
+            $c->stash->{contacts},
+            # The cobrand may wish to provide its own list of bodies independent
```

Comment:
Does this not apply the other way also? Why would London borough cobrands want to see TfL response templates?